### ISSUE TITLE
Melpa

### ISSUE THREAD

**Issue by grolongo** [atheriel/helm-twitch#2]
Hey, I just watched your EmacsConf presentation and I was pretty happy with your demo!

As an avid twitch user myself, I was impressed on how you integrated it into Emacs, it seems very smooth.

Unfortunately I only use Ivy/Counsel so I will keep using Chatty with <s>Livestreamer</s> streamlink and mpv.

Will you add your package to Melpa?

Anyway good job! 👍

**Comment by atheriel** [comment]
Yes, I think I should start the process of releasing on MELPA, likely as a few different packages.

> Unfortunately I only use Ivy/Counsel so I will keep using Chatty with Livestreamer streamlink and mpv.

Which parts of this project will you make use of in combination with Ivy/Counsel?

**Comment by grolongo** [comment]
> Which parts of this project will you make use of in combination with Ivy/Counsel?

Probably none unless someone makes an implementation for Ivy. Wish I could help but my Elisp skills are very... limited.

I might try it even with Helm though just to see how it looks and tinker a bit!

**Comment by ghost** [comment]
I would love to see this get put on MELPA. Any status on that?
